Formula/ncrack.rb

```diff
@@ -1,7 +1,9 @@
 class Ncrack < Formula
   desc "Network authentication cracking tool"
   homepage "https://nmap.org/ncrack/"
-  license "GPL-2.0-only"
+  # The program license includes other non-representable exceptions in addition
+  # to the OpenSSL exception. These can be seen in the installed COPYING file.
+  license "GPL-2.0-only" => { with: "openvpn-openssl-exception" }
```

Comment:
Not sure if better to go with this or `:cannot_represent`.

All exceptions must be accepted https://github.com/nmap/ncrack/blob/master/COPYING#L9-L12
```
 * This program is free software; you may redistribute and/or modify it    *
 * under the terms of the GNU General Public License as published by the   *
 * Free Software Foundation; Version 2 ("GPL"), BUT ONLY WITH ALL OF THE   *
 * CLARIFICATIONS AND EXCEPTIONS DESCRIBED HEREIN.  This guarantees your   *
```

OpenSSL exception https://github.com/nmap/ncrack/blob/master/COPYING#L60-L64
```
 * As another special exception to the GPL terms, the Nmap Project grants  *
 * permission to link the code of this program with any version of the     *
 * OpenSSL library which is distributed under a license identical to that  *
 * listed in the included docs/licenses/OpenSSL.txt file, and distribute   *
 * linked combinations including the two.                                  *
```

---

Fedora just refers to this as **with exceptions** (https://src.fedoraproject.org/rpms/ncrack/blob/rawhide/f/ncrack.spec#_8)

Debian just refers to this as **with nmap exception** (https://metadata.ftp-master.debian.org/changelogs//main/n/ncrack/ncrack_0.7+debian-3_copyright)

Neither is a part of SPDX.

Reply:
@Homebrew/plc?

Reply:
I'd say `:cannot_represent`, clearly without our SPDX requirement it's already hard to represent.

Reply:
Agreed on `:cannot_represent`.